Make appointment functionality added

coolfortune opened the pull request:
I made a page that's essentially a form for tutors to create and schedule new appointments. The reason I made the profile page build into this page is purely for development purposes, which is why I made the page its own custom widget.

coolfortune committed:
Make appointment functionality added

aguilarjavier commented:
Dude the date and time inputs are sick. I really really like them. When choosing class code though my screen just faded a bit. Not sure what happened there. Seems like its trying to display something but isn't. Also is there a way to limit the time to just hours? That way appointments are made in hour intervals and easier for users to remember.
Overall great job.
All hail Jeff

Tyiii reviewed (approved):
Bro good job man! this took more than i thought for sure.

coolfortune committed:
Color UI changes in the Date and Time picker
Adds the time, date, and tutorName field in the database to avoid assert errors

tjinks89 reviewed (approved):
Everything is coming together well, keep up the good work man.

I was having an issue with the class selection as well...other than that, I like it.
